DavidDowell/pul-request-demo
# add text to README.md

## To test
-Open index.html and press the increment button
-verify that the count updates by 1 and the Ui is updated as well.
